### DISCUSSION TITLE
Multiple Discord Agents

### DISCUSSION THREAD

**Discussion by Bovive** [sct/overseerr#3590] [Feature Requests]
Perhaps I am doing this incorrectly and it is possible. I prefer to use a private notification channel for request notifications, issues, etc. I would like to be able to have approvals (with mentions) sent to a separate channel that the requesting users have access to, which I understand would be a separate webhook. I know I can use a separate notification service for my purposes, but I was curious if this is possible. Thanks!

**Comment by marchershey** [comment]
I use Tautulli for that, and it does just that. There's a whole list of triggers you can set for a specific channel. 

The discord channel I use for Overseerr is only for "media-issues" only, the rest are from Tautulli. 

![image](https://github.com/sct/overseerr/assets/2856490/918c4179-4727-4ef8-b9a0-9a4603289ecb)

![image](https://github.com/sct/overseerr/assets/2856490/f8633853-4d9f-415c-9f22-a97fa575e0d0)

![image](https://github.com/sct/overseerr/assets/2856490/0435edf8-a960-43bf-b681-d662e050cc6a)

![image](https://github.com/sct/overseerr/assets/2856490/8b0dc9f2-f69e-4ec6-8560-68cfa99d3283)

![image](https://github.com/sct/overseerr/assets/2856490/1061a50b-6a50-453e-9a9e-f840e625d1e6)

**Reply by Piddox** [reply]
Tautulli is great, but can't handle requests. I would like to send out request notifications to more than 1 channel. It would be great if Overseerr copies the idea of Tautulli and enables multiple discord agents (or any agent really).
